Autopilot/Inc/airspeed.hpp

```diff
@@ -0,0 +1,59 @@
+/**
+*   
+*   Airspeed sensor functions
+*
+*   This header is an outline of basic functions of the airspeed
+*   sensor. Generally, this sensor is a Pitot tube which uses a 
+*   pressure differential to determine airspeed. 
+*
+*   Airspeed calculation should be done and returned in meters 
+*   per second
+*
+*    Author: Annie LePage
+**/
+
+
+#ifndef AIRSPEED_HPP
+#define AIRSPEED_HPP
+
+#include <stdint.h>
+/*
+    Currently there is only one airspeed sensor used, if this is 
+    changed to a different sensor, part numbers and a selection
+    option should be added
+*/
+
+struct airspeedData_t 
+{
+    double airspeed;    // in m/s
+
+    float utcTime;      // 4 Bytes. Time in seconds since 00:00 
+};
+
+class airspeed 
+{
+    public:
+        /**
+         *  Initializes ADC in order to recieve sensor readings 
+         *  (pitot tube reading)
+         * */
+        virtual void Init(); 
```

Comment:
Had some discussion with Lucy about this. We might aswell make these pure virtual methods, since it's *slightly* safer and more explicit. https://www.youtube.com/watch?v=ndz3EHpFEZc&t=46s is a good explanation of virtual vs pure virtual.

Pure virtual would look like:
virtual void Init() = 0;

Reply:
Sounds good